R/out_MainContent.R

```diff
@@ -0,0 +1,26 @@
+out_MainContent <- function(strTitle = "GSM Deep Dive") {
+  div(
+    id = "main-content",
+    fluidRow(
+      out_TopSpacer(),
+      # For now, we're intentionally only using 10/12 of this space, ie one
+      # "column" that's 10 columns wide.
+      #
+      # TODO: This should be handled via CSS.
+      column(
```

Comment:
My intention is to continue this update in another PR, switching entirely to `bslib::page_fillable()` (and other fillable things down from there). That's reflected in #208, but moreso in a larger ticket I'm in the middle of writing about an overall UI polish. I'm resisting the urge to do more right now because this PR has already grown way beyond what I intended!